### ISSUE TITLE
Issue with RiotGames.RiotClient (#16141)

### ISSUE THREAD

**Issue by Rerago** [microsoft/winget-pkgs#16349]
# Brief description of your issue

### Problem 1

The version is not determined and what in turn is a problem with ``winget upgrade --all``. And because of problem 2 is somewhat unpleasant. But this was already mentioned in #16287.

### Problem 2: 

Riot/League of Legends has different servers (EUW, NA and so on). But in #16141 the NA client was used. This leads to problems with an EUW installtion because a) the language and b) the server are changed.

### What would need to change?

Riot would have to provide an installer that a) sets the version in the software center and b) is independent of the server and does not change the settings. Or it would have to provide different installers (RiotGames.RiotClient.NA, RiotGames.RiotClient.EUW and so on).

# Steps to reproduce

1. install: https://lol.secure.dyn.riotcdn.net/channels/public/x/installer/current/live.euw.exe
2. run ```winget upgrade --all```

# Expected behavior

The update is performed only once and does not change the server and language settings.

# Actual behavior

The update is executed again each time ``winget upgrade --all`` is called and changes the language and server settings each time.

# Environment

```
Windows Package Manager v1.0.11451

Windows: Windows.Desktop v10.0.19043.1023
Package: Microsoft.DesktopAppInstaller v1.11.11451.0
```

**Comment by lychichem** [comment]
For problem 2,maybe we need to write a multi-locale yaml file which use different installer for different locale.

**Comment by Rerago** [comment]
I don't think this will help. Since the installer is server based rather than language based. 
(In my example, the language was only changed because many of the European languages are not available for selection on the NA server (for whatever reason)).

So someone from the USA (NA) could also play on a EUW (EU-West) server (even if it makes little sense for ping reasons). But a realistic example would be Poland. There is also EUNE (EU-North-East). Whether someone from Poland plays on EUW or EUNE is up to the player and cannot really be determined by the language "pl-PL" (especially since it is common to play in English).

**Comment by Rerago** [comment]
@ivey @riot-rmarin

I know it's definitely not your area/department. But can you forward this to the position/colleague who could be helpful here?

**Comment by OfficialEsco** [comment]
Duplicate of #24222
